include/picongpu/param/ionizationEnergies.param

```diff
@@ -137,6 +137,24 @@ namespace AU
         2304.14 * UNITCONV_eV_to_AU
     );
 
+    /* ionization energy for silicon in atomic units */
+    PMACC_CONST_VECTOR(float_X, 14, Silicon,
+        8.151683 * UNITCONV_eV_to_AU,
```

Comment:
This has nothing to do with your PR, just a side note for future self: these constants are double, should be changed to `..._X` to be `float_X`.